### PR TITLE
Add Kafka CDI library dependency [AGPUSH-2120]

### PR DESCRIPTION
**Pull request by polinankoleva**

### CODE REVIEW

**Comment by polinankoleva** [comment]
@ziccardi @dimitraz review this

**Comment by dimitraz** [comment]
Great 👍

File: pom.xml [hunk 1]
```diff
@@ -199,6 +199,7 @@
         <aerogear.bom.version>0.2.20.2</aerogear.bom.version>
 
         <kafka.clients.version>0.10.2.1</kafka.clients.version>
+        <kafka.cdi.version>0.0.4</kafka.cdi.version>
```
Comment (dimitraz) [inline pom.xml]: Can we upgrade this to the latest version, `0.0.5`

Comment (polinankoleva) [inline pom.xml]: Good point!

**Comment by ziccardi** [comment]
LGTM

**Comment by polinankoleva** [comment]
I think this can be merged if everyone approves @ziccardi

**Comment by ziccardi** [comment]
Conflicts must be fixed

**Comment by polinankoleva** [comment]
@ziccardi conflicts resolved.

**Pull request merged**